README.md

```diff
@@ -27,32 +27,6 @@ The source code for the project is contained in this mono-repo. Within the [pack
 
 ## Running the project
 
-### Using kubernetes
-
-#### Secrets
-
-The following kubernetes secrets are used:
-
-    DRIVER_TOKEN - Driver Telegram bot token. Used by driver-interace
-    GOOGLE_TOKEN - Used by driver-interace
-    POSTNORD_KEY - Used by engine-server to get information from Postnord API
-    MINIO_ROOT_PASSWORD - Used by minio and engine-server
-    POSTGRES_PASSWORD - Used by postgres, engine and postgres-backup
-
-#### Deployment
-
-To deploy the dependencies of the stack (usually done once and it's DBs) to your Kubernetes cluster, use Skaffold:
-
-    skaffold -f skaffold-dependencies.yaml run
-
-To deploy the relevant packages to your Kubernetes cluster, use Skaffold:
-
-    skaffold run --tail
-
-To debug run:
-
-    skaffold dev
-
 #### Before you start running the project
 
 Start by exporting port 9200
```

Comment:
Why do you need to do this?

Reply:
Expose the elasticsearch port that is

Reply:
@iteadam 
I don't know all details about this and it's part of `Before you start running the project` which applies to localhost.
This PR documents the infrastructure in the "cloud".

I think it's something with signatures, image recognition, driver interface...I am actually not that familiar with this.
But I still don't think this information relating running the project locally should block the PR.

Reply:
no I was just looking through and went "wtf is this, I've never done this"

Reply:
I might have done it once and I think it's related to scanning shipment labels when you run the project locally.